salt/utils/win_runas.py

```diff
@@ -127,6 +128,12 @@ def runas(cmd, username, password=None, cwd=None):
         impersonation_token = None
         win32api.CloseHandle(th)
 
+    if isinstance(cmd, (list, tuple)):
+        # CreateProcess parameter lpCommandLine must be a string.
+        # Since it is called directly and not via the subprocess module,
+        # the arguments must be processed manually.
+        cmd = subprocess.list2cmdline(cmd)
```

Comment:
should this codeblock be in `runas_unpriv` as well?

Reply:
`runas_unpriv` is only referenced in `runas`, after subprocess.list2cmdline() is called.

Reply:
True